docs/source/MaxSATDecoder.rst

```diff
@@ -0,0 +1,26 @@
+MaxSAT Decoder
+=============
+
+The MaxSAT package provides a MaxSAT decoder for decoding quantum codes. Currently the automatic simulation of
+triangular color codes is supported. The decoder is based on the Z3 MaxSAT solver.
+
+Usage
+#####
+
+The decoder is implemented in :func:`~mqt.qecc.cc_decoder` and simulations can be run automatically with the run method:
```

Comment:
`mqt.qecc.cc_decoder` is not a function and will most likely not be rendered correctly.
On the other hand, `run` is a function that could be directly linked to. 
The relevant function should be added to the API docs.